desktop-app/src/main/menu/view.ts

```diff
@@ -32,9 +32,9 @@ const getReloadMenu = (
   click: () => {
     if (isDev) {
       mainWindow.webContents.reload();
-      return;
+      // return;
     }
-    mainWindow.webContents.send('reload', {});
+    // mainWindow.webContents.send('reload', {});
```

Comment:
Why reason why this is commented out?

Reply:
Ok, but if we have this commented then the menu option, won't work, I guess? 
The `View` menu -> `Reload` option.

Reply:
Oops! Apologies for missing that. I will make the necessary changes. 😅